.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 pkg/*.gem
+Gemfile.lock
```

Comment:
Sure, as you wish :)
This `Gemfile.lock` just doesn't work for me with ruby 2.2.0, I'll return it back